docs/servo_power_monitor/src/index.ts

```diff
@@ -96,6 +95,20 @@ form.addEventListener('submit', async e => {
   }
 });
 
+input.addEventListener('keydown', async e => {
+  if (DUTPort === undefined) {
+    overlay.classList.remove('closed');
+  } else {
```

Comment:
instead of using else, you can use "early return" to reduce the levels of nesting. so it will looks like:
```
  if (DUTPort === undefined) {
    overlay.classList.remove('closed');
    return;
  }
  const DUTWritable = DUTPort.writable;
```

Reply:
I fixed it!